core/src/test/java/com/zextras/carbonio/files/message_broker/UserStatusChangedIT.java

```diff
@@ -1,128 +0,0 @@
-// SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
-//
-// SPDX-License-Identifier: AGPL-3.0-only
-
-package com.zextras.carbonio.files.message_broker;
-
-import com.google.inject.Injector;
-import com.zextras.carbonio.files.Simulator;
-import com.zextras.carbonio.files.Simulator.SimulatorBuilder;
-import com.zextras.carbonio.files.api.utilities.DatabasePopulator;
-import com.zextras.carbonio.files.api.utilities.entities.SimplePopulatorTextFile;
-import com.zextras.carbonio.files.dal.dao.ebean.Node;
-import com.zextras.carbonio.files.dal.repositories.interfaces.NodeRepository;
-import com.zextras.carbonio.files.message_broker.consumers.UserStatusChangedConsumer;
-import com.zextras.carbonio.message_broker.events.services.mailbox.UserStatusChanged;
-import com.zextras.carbonio.message_broker.events.services.mailbox.enums.UserStatus;
-
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
-
-import org.assertj.core.api.Assertions;
-import org.assertj.core.api.Condition;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Test;
-
-/**
- * Here I mock an event emitted and received because if I was to emit a real event I could not know
- * when it will be consumed
- */
-class UserStatusChangedIT {
```

Comment:
Why is this test being removed?

Reply:
@federicorispo It has been copied from .../message_broker/UserStatusChangedIT.java to .../message_broker/it/UserStatusChangedIT.java without removing the file on the original position in a previous feat/fix, so there were two exact copies of the test. I deleted the original one with this feature.